requirements/requirements.txt

```diff
@@ -0,0 +1,11 @@
+RPi.GPIO~=0.7
+smbus2~=0.4
+ovos_utils~=0.0.25
+board~=1.0
+click~=8.0
+click-default-group~=1.2
+rpi_ws281x~=4.3
+adafruit-circuitpython-neopixel~=6.3
+adafruit-blinka~=8.6
+spidev~=3.5
+smbus~=1.1
```

Comment:
This fixed errors with boot animation, but it shouldn't be necessary. Troubleshoot removal before first release